src/Parsers/MarkupParser.php

```diff
@@ -98,7 +98,9 @@ public function getTerms()
         $cleanMarkup = strip_tags(strtolower($this->getMarkup()));
 
         // Count the output and sort from highest to lowest.
-        $words = array_count_values(str_word_count($cleanMarkup, 1));
+        $namePattern = '/[\s,:?!]+/u';
```

Comment:
Sorry - I'm just playing this through in my head, but shouldn't the `?` be escaped as well? Or is it fine in a capture class?